plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -1715,4 +1715,23 @@ private void onCassandra(@Language("SQL") String sql)
     {
         session.execute(sql);
     }
+
+    @Test
+    public void testExecuteProcedure()
+    {
+        try (TestCassandraTable table = testTable(
+                "execute_procedure",
+                ImmutableList.of(partitionColumn("key", "int")),
+                ImmutableList.of())) {
+            String keyspaceAndTable = table.getTableName();
+            assertUpdate("CALL system.execute('INSERT INTO " + keyspaceAndTable + " (key) VALUES (1)')");
+            assertQuery("SELECT * FROM " + keyspaceAndTable, "VALUES 1");
+
+            assertUpdate("CALL system.execute('DELETE FROM " + keyspaceAndTable + "')");
+            assertQueryReturnsEmptyResult("SELECT * FROM " + keyspaceAndTable);
+
+            assertUpdate("CALL system.execute('DROP TABLE " + keyspaceAndTable + "')");
+            assertThat(getQueryRunner().tableExists(getSession(), keyspaceAndTable)).isFalse();
```

Comment:
Remove these lines. Dropping table here causes an exception at `TestCassandraTable.close()`. 
Note that `tableExists` method expects a table name without the schema name.

Reply:
> Remove these lines. Dropping table here causes an exception at `TestCassandraTable.close()`. Note that `tableExists` method expects a table name without the schema name.

done.